components/brave_adaptive_captcha/DEPS

```diff
@@ -0,0 +1,4 @@
+include_rules = [
+  "+content/public/browser",
```

Comment:
see https://github.com/brave/brave-core/pull/9350/files#r694133600

Reply:
Fixed, thank you.

Reply:
Actually, I'm a little confused why we can't have this dependency when services use `content::BrowserContext`.